src/vscode-dts/vscode.proposed.chatStatusItem.d.ts

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
+
+	export interface ChatStatusItem {
+		/**
+		 * The identifier of this item.
+		 */
+		readonly id: string;
+
+		/**
+		 * The main name of the entry, like 'Indexing Status'
+		 */
+		title: string;
+
+		/**
+		 * Optional additional description of the entry.
+		 *
+		 * This is rendered less prominent on the same line as the title. Supports Markdown style links and rendering of
+		 * {@link ThemeIcon theme icons} via the `$(<name>)`-syntax.
+		 */
+		description: string | undefined;
+
+		/**
+		 * Optional additional details of the entry.
+		 *
+		 * This is rendered less prominent in a separate line. Supports Markdown style links and rendering of
+		 * {@link ThemeIcon theme icons} via the `$(<name>)`-syntax.
+		 */
+		details: string | undefined;
+
```

Comment:
Curious why we wouldn't support `MarkdownString` here and just `string`. I am comparing this to the `tooltip` property of status bar items.

Reply:
nevermind, I think this is better to limit how noisy this visually will look like